source/grid/grid_tools.cc

```diff
@@ -5851,6 +5851,63 @@ namespace GridTools
 
 
 
+    template <int dim, int spacedim>
+    std::tuple<std::vector<unsigned int>,
+               std::vector<unsigned int>,
+               std::vector<unsigned int>>
+    guess_point_owner(
+      const parallel::distributed::Triangulation<dim, spacedim> &tria,
+      const std::vector<Point<spacedim>> &                       points,
+      const double                                               tolerance)
+    {
+      (void)tolerance;
```

Comment:
@konsim83 Is there a possibility to specify a tolerance?

Reply:
The answer is (hopefully) in the previous thread :-)